vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -95,12 +108,19 @@ def build(self):
             fluid=True,
         )
 
-    def _make_page_layout(self, page: Page):
+    def _get_page_divs(self, page: Page) -> PageDivs:
         # Identical across pages
+        # TODO: Implement proper way of automatically pulling file called logo in assets folder (should support svg, png and it shouldn't matter where it's placed in the assets folder)
+        logo = (
+            html.Div([html.Img(src=get_asset_url("logo.svg"), className="logo-img")], className="logo", id="logo")
+            # TODO: Implement condition check if image can be found/not found
+            if True
+            else html.Div(id="logo", hidden=True)
+        )
```

Comment:
@nadijagraca - this part is what I am leaving up to you. It's hard-coded here just so I could test a few things, but in general that True/False condition would have to be detected automatically. See the relevant TODO :)

Reply:
This is worth discussing first @nadijagraca because @AnnMarieW will need something very similar for her work on meta tags - see #176. We should end up with something similar to [Dash's `_infer_images`](https://github.com/plotly/dash/blob/dev/dash/_pages.py#L24) but implemented in Pathlib, which makes it very easy to search recursively (see our own `Vizro.__get_external_assets`).

Reply:
Just FYI - I''m making progress on retrieving the images for the logo, meta tags, and the favicon from the app's assets folder. :slightly_smiling_face:

Reply:
Amazing - thanks @AnnMarieW ! Let me know if you need any help or we should review :) The icon part will be removed from this PR in the mean time